minerva-lookup/src/main/java/org/geneontology/minerva/lookup/MonarchExternalLookupService.java

```diff
@@ -58,7 +58,7 @@ protected MonarchExternalLookupService(String golrUrl, RetrieveGolrOntologyClass
 				if (entities != null && !entities.isEmpty()) {
 					result = new ArrayList<ExternalLookupService.LookupEntry>(entities.size());
 					for(GolrOntologyClassDocument doc : entities) {
-						result.add(new LookupEntry(id, doc.annotation_class_label, "ontology_class", null));
+						result.add(new LookupEntry(id, doc.annotation_class_label, "ontology_class", null, doc.isa_partof_closure));
```

Comment:
@goodb shouldn't this be only `isa_closure`, not `isa_partof_closure`? (same question as in other file)